docs/sql-migration-guide.md

```diff
@@ -83,6 +83,8 @@ license: |
 
   - In Spark 3.2, the unit-to-unit interval literals like `INTERVAL '1-1' YEAR TO MONTH` are converted to ANSI interval types: `YearMonthIntervalType` or `DayTimeIntervalType`. In Spark 3.1 and earlier, such interval literals are converted to `CalendarIntervalType`. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.interval.enabled` to `true`.
 
+  - In Spark 3.2, Spark supports `DayTimeIntervalType` and `YearMonthIntervalType` as inputs and outputs of `TRANSFORM` clause in Hive `SERDE` mode, the behavior is different between Hive `SERDE` mode and `ROW FORMAT DELIMITED` mode when this two types are used as inputs. In Hive `SERDE` mode, `DayTimeIntervalType` column is converted to `HiveIntervalDayTime`, it's string format is `[+|-]?d h:m:s.n`, but in `ROW FORMAT DELIMITED` mode the format is `INTERVAL [-|+]?'[-|+]?d h:m:s.n' DAY TO TIME`. In Hive `SERDE` mode, `YearMonthIntervalType` column is converted to `HiveIntervalYearMonth`, it's string format is `[+|-]?y-m`, but in `ROW FORMAT DELIMITED` mode the format is `INTERVAL [-|+]?'[-|+]?y-m' YEAR TO MONTH`.
```

Comment:
`it's string format is `[+|-]?d h:m:s.n` -> `its string format is `[+|-]?d h:m:s.n``

Reply:
it's string format is `[+|-]?y-m` -> its string format is `[+|-]?y-m`

Reply:
DOne